app/views/payment_tracking/index.html.erb

```diff
@@ -45,7 +45,7 @@
         <tr>
           <td><input type="checkbox" id=<%= "paymentTrackingCheck#{rental.id}" %> ></td>
           <td><%= rental.id %></td>
-          <td><%= rental.reservation_id %></td>
+          <td><%= rental.str_reservation_ids.truncate(77) %></td>
```

Comment:
Why 77?

Reply:
that is two uuid's separated by a space and a comma with 3 extra characters for the ... to show truncation.

Reply:
Okay. Since all of that logic is encoded in your method, I'd suggest having that truncation be an option you can specify in the method arguments.